### ISSUE TITLE
Broken dark mode diagram with diagram inside collapsible admonition

### ISSUE THREAD

**Issue by smartYSC** [MikhailKravets/mkdocs_puml#64]
**Describe the bug**
If a diagram is put into a collapsible admonition, the light mode diagram will be in the admonition (and collapsed). When switching to dark mode the diagram will be below the admonition and the admonition is empty.

**To Reproduce**

```markdown
??? info "Broken Admonition"

    ```plantuml
    @startuml
    Alice -> Bob: test
    @enduml
    ```
```

**Expected behavior**
Diagram is inside admonition, not below it.


**Screenshots**
Light mode is correct.
<img width="589" alt="Light Mode" src="https://github.com/user-attachments/assets/91ab8b7a-1678-452b-a744-1e9d5fa34258">

Dark mode is broken.
<img width="589" alt="Dark Mode" src="https://github.com/user-attachments/assets/7c38e3a8-43ad-4d1d-a7f0-fc9d0e8448ae">


**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 LTS in WSL2
 - Dependency versions: `mkdocs==1.6.1` `mkdocs_puml==2.0.0` `mkdocs-material==9.5.40`

**Comment by MikhailKravets** [comment]
Thanks for letting me know about this @smartYSC. It's interesting that only one of them is broken — I thought both should be! 😅

**Comment by MikhailKravets** [comment]
The problem was in `\n` characters. Markdown replaces them as `<p>` tag that produces this behavior. Anyway it's fixed in `2.1.0` version.

**Comment by smartYSC** [comment]
Can confirmed this is resolved, thanks a lot for the quick fix!
